dashboard/src/t5gweb/libtelco5g.py

```diff
@@ -207,6 +250,9 @@ def create_cards(cfg, new_cases, action='none'):
     component = get_component_id(jira_conn, project.id, cfg['component'])
     board = get_board_id(jira_conn, cfg['board'])
 
+    # Obtain the authentication token for RedHat Api
+    token = get_token()
```

Comment:
should be `get_token(cfg['offline_token'])`

Reply:
I'll fix it.